niri-ipc/src/lib.rs

```diff
@@ -578,6 +578,14 @@ pub enum Action {
         )]
         y: PositionChange,
     },
+    /// Toggles the opacity of the focused window
+    ToggleWindowOpacity {
```

Comment:
So I'm thinking, this is a bit confusing, because windows can have their own client-side opacity, and this action doesn't (and shouldn't) toggle that. Maybe it's better to call it `ToggleWindowRuleOpacity` or something?